### PR TITLE
Upgrade xorm to v1.0.0

### PR DESCRIPTION
**Pull request by lunny**
As title.

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -787,9 +787,7 @@ xorm.io/builder v0.3.6 h1:ha28mQ2M+TFx96Hxo+iq6tQgnkC9IZkM6D8w9sKHHF8=
...
 xorm.io/core v0.7.2 h1:mEO22A2Z7a3fPaZMk6gKL/jMD80iiyNwRrX5HOv3XLw=
 xorm.io/core v0.7.2/go.mod h1:jJfd0UAEzZ4t87nbQYtVjmqpIODugN6PD2D9E+dJvdM=
-xorm.io/core v0.7.3 h1:W8ws1PlrnkS1CZU1YWaYLMQcQilwAmQXU0BJDJon+H0=
-xorm.io/core v0.7.3/go.mod h1:jJfd0UAEzZ4t87nbQYtVjmqpIODugN6PD2D9E+dJvdM=
 xorm.io/xorm v0.8.0 h1:iALxgJrX8O00f8Jk22GbZwPmxJNgssV5Mv4uc2HL9PM=
 xorm.io/xorm v0.8.0/go.mod h1:ZkJLEYLoVyg7amJK/5r779bHyzs2AU8f8VMiP6BM7uY=
-xorm.io/xorm v0.8.2-0.20200120024500-c37aff9b3a4a h1:hzGd080rlkZ5a7v6Tr3x8PJJnWPfKxGMMl92c8DNcww=
-xorm.io/xorm v0.8.2-0.20200120024500-c37aff9b3a4a/go.mod h1:ZkJLEYLoVyg7amJK/5r779bHyzs2AU8f8VMiP6BM7uY=
+xorm.io/xorm v0.8.3-0.20200306120832-0f166d82da7f h1:+QaddYm7KvKVNYZifXzAMTsEVTGKYmhezMCmAPQs5k8=
+xorm.io/xorm v0.8.3-0.20200306120832-0f166d82da7f/go.mod h1:zylddI/y190bUT26wbSjzDrDnZzs6Gxq33nY2eL2VXI=
```
Comment (techknowlogick) [inline go.sum]: Is title wrong, this is `xorm.io/xorm v0.8.3` not `1.0.0`?

Comment (lunny) [reply go.sum]: @techknowlogick I haven't tagged xorm v1.0.0 yet, just want to have a test on gitea.

**Comment by 6543** [comment]
@lunny this will also fix some tests: https://github.com/go-gitea/gitea/pull/10520/commits/ccaa6ba78170b93dbaba2585a6ce4726330f834b

**Comment by 6543** [comment]
this pull will close #10520 too

File: routers/admin/auths.go [hunk 2]
```diff
@@ -18,9 +18,9 @@ import (
 	"code.gitea.io/gitea/modules/log"
 	"code.gitea.io/gitea/modules/setting"
 	"code.gitea.io/gitea/modules/util"
+	"xorm.io/xorm/convert"
```
Comment (lafriks) [inline routers/admin/auths.go]: Wrong order

**Review by lafriks — APPROVED** [review]

**Review by techknowlogick — APPROVED** [review]

**Pull request merged**